### ISSUE TITLE
Dchange not working for keys with double quotes

### ISSUE THREAD

**Issue by dinuka10** [wso2/api-manager#129]
**Description:**
Dchange not working for keys with double quotes

**Steps to reproduce:**

1.)  Add below key and password in the secret section of deployment.toml file in MI.
 [secrets]
"bally.api.rsa.key" = "[example_password]"

2.) Now go to the bin directory and execute " ./ciphertool.sh -Dconfigure" command. Then you will notice that example_password is encrypted.

3.) Now go to the bin directory again and execute ""./ciphertool.sh Dchange" (Please refer - Dchange.png image)  (Restart the server or start and stop the server before you run this command)
![Dchange](https://user-images.githubusercontent.com/44248646/171101726-878577f2-e055-4896-978e-64c8d4977ede.png)

4.) Then you will notice that Ciphertext of the password has not changed ( Password not changed)

When I analyzed this behavior , I found that the cause for this behavior is double quotes in the key ( Ex - *"bally.api.rsa.key") .If i use bally.api.rsa.key (*Without double quotes) as a key instead of "bally.api.rsa.key" , Dchange worked as expected ( cipertext was changed).

[secrets]
bally.api.rsa.key = "[example_password]"

But if I use this key without double quotes (Like above) ,  "wso2:vault-lookup('bally.api.rsa.key')" 
is not working as expected .

So in summary , if we use double quotes in a key Dchange in ciper tool is not working .
If we use a key without double quotes wso2:vault-lookup is not working

**Comment by hisanhunais** [comment]
The fix has been done via [1] and is available from cipher tool version 1.2.1 [2] onwards. Need to upgrade the product-apim cipher tool dependency version

[1] https://github.com/wso2/cipher-tool/pull/74
[2] https://github.com/wso2/cipher-tool/tree/v1.2.1
